vipps-order-management-api-checklist.md

```diff
@@ -0,0 +1,77 @@
+<!-- START_METADATA
+---
+title: Checklist
+sidebar_position: 50
+---
+END_METADATA -->
+
+# Vipps Order Management API Checklist
+
+<!-- START_COMMENT -->
+
+ℹ️ Please use the new documentation:
+[Vipps Technical Documentation](https://vippsas.github.io/vipps-developer-docs/).
+
+<!-- END_COMMENT -->
+
+API version: 1.0.
+
+## Checklist
+
+- [ ] Integrate _all_ the [API endpoints][order-mgmt-api-reference-url]:
```

Comment:
Hm, this is not always necessary. It is fine for merchants to only use /receipts or /categories if they dont want both :)